source/extensions/health_checkers/redis/config.cc

```diff
@@ -0,0 +1,34 @@
+#include "extensions/health_checkers/redis/config.h"
+
+#include "envoy/registry/registry.h"
+
+#include "common/config/utility.h"
+
+#include "extensions/health_checkers/redis/utility.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HealthCheckers {
+namespace RedisHealthChecker {
+
+Upstream::HealthCheckerSharedPtr RedisHealthCheckerFactory::createCustomHealthChecker(
+    const envoy::api::v2::core::HealthCheck& config,
+    Server::Configuration::HealthCheckerFactoryContext& context) {
+
+  return std::make_shared<RedisHealthChecker>(
+      context.cluster(), config, getRedisHealthCheckConfig(config), context.dispatcher(),
+      context.runtime(), context.random(),
+      Extensions::NetworkFilters::RedisProxy::ConnPool::ClientFactoryImpl::instance_);
```

Comment:
nit: You can remove `Extensions::` prefix as you are already in Extensions namespace.